### ISSUE TITLE
📝 CHANGES.md is stale

### ISSUE THREAD

**Issue by pavelpikatamcs** [cloudflare/cloudflared#1287]
CHANGES.md no longer contains up-to-date information about new versions and what has changed. Can this be updated please.

**Comment by nikolay** [comment]
I agree! The changelog file either needs to be kept up-to-date or deleted.
